kitsune/products/jinja2/products/product.html

```diff
@@ -81,7 +81,7 @@ <h1 class="product-title cf">
   <div class="grid_12">
     <section id="help-topics">
       <h1>{{ _('Help Topics') }}</h1>
-      {{ help_topics(topics, show_community_support=product.questions_enabled(request.LANGUAGE_CODE)) }}
+      {{ help_topics(topics, show_community_support) }}
```

Comment:
As The default value of ``show_community_support`` is ``False`` [here](https://github.com/mozilla/kitsune/blohttps://github.com/mozilla/kitsune/blob/a8ed6f598293be6d7286cae2bf8ae3f4d9ea7d8b/kitsune/products/jinja2/products/includes/topic_macros.html#L1b/a8ed6f598293be6d7286cae2bf8ae3f4d9ea7d8b/kitsune/products/jinja2/products/includes/topic_macros.html#L1), So until you pass the value through it, it will use the default value.